File: bindings/java/examples/binding_surface9InteractiveDocx.java
package io.wellfriendpdf.examples;

import java.nio.file.Files;
import java.nio.file.Path;
import io.wellfriendpdf.Wellfriend;

public final class FormActionPolicyInteractiveDocx {
    public static void main(String[] args) throws Exception {
        if (args.length != 1) {
            throw new IllegalArgumentException("usage: FormActionPolicyInteractiveDocx input.pdf");
        }
        try (WellfriendPdf.Document document = WellfriendPdf.Document.open(Path.of(args[0]))) {
            System.out.println(document.formJavaScriptReportJson());
            System.out.println(document.interactiveDataReportJson());
            System.out.println(document.form_action_policyReportJson());
            Files.write(
                Path.of("form_action_policy-page-faithful.docx"),
                document.toDocx("page-faithful", true));
        }
    }
}

File: bindings/dotnet/examples/binding_surface9InteractiveDocx.cs
using WellfriendPdf;

if (args.Length != 1)
{
    Console.Error.WriteLine("usage: FormActionPolicyInteractiveDocx input.pdf");
    return 2;
}

using var document = WellfriendDocument.Open(File.ReadAllBytes(args[0]));
Console.WriteLine(document.FormJavaScriptReportJson());
Console.WriteLine(document.InteractiveDataReportJson());
Console.WriteLine(document.FormActionPolicyReportJson());
File.WriteAllBytes("form_action_policy-page-faithful.docx", document.ToDocx("page-faithful"));
return 0;
